### PR TITLE
Abuse report viewer fixes (bug 1165881 & more)

### PR DESCRIPTION
**Pull request by eviljeff**
fixes bug 1165881 and makes it work with website abuse reports (hadn't landed for the first patch)

### CODE REVIEW

**Comment by eviljeff** [comment]
no idea why the test is failing - the failure is on the delete() itself

**Comment by diox** [comment]
We need to adjust the ES mock in our base `TestCase`. Currently it patches:
```
 ES_patchers = [mock.patch('elasticsearch.Elasticsearch'),
                mock.patch('mkt.webapps.indexers.WebappIndexer', spec=True),
                mock.patch('mkt.search.indexers.index', spec=True),
                mock.patch('mkt.search.indexers.Reindexing', spec=True,
                           side_effect=lambda i: [i])]
```

We should either add `'mkt.websites.indexers.WebsiteIndexer'` to that list (easiest) or change the first line to patch `'mkt.search.indexers.BaseIndexer'` instead (cleanest, but not sure it'd work - might break other tests, needs to be tested)

File: mkt/reviewers/tests/test_views.py [hunk 1]
```diff
@@ -3392,6 +3393,101 @@ def test_deleted_app(self):
         eq_(doc('.tabnav li a')[0].text, u'Abuse Reports (1)')
 
 
+class TestWebsiteAbuseQueue(mkt.site.tests.TestCase, AccessMixin):
```
Comment (diox) [inline mkt/reviewers/tests/test_views.py]: I didn't think about that when you did the app abuse viewer, but can you add XSS tests ? To make sure we are properly escaping the data displayed in the queue etc.

**Comment by diox** [comment]
r+wc

**Pull request merged**